circus-train-core/src/test/java/com/hotels/bdp/circustrain/core/replica/hive/RenameTableOperationTest.java

```diff
@@ -55,8 +55,8 @@ public void setUp() throws TException {
     toTable.setTableName(TO_TABLE_NAME);
     fromTable.setDbName(FROM_DB_NAME);
     toTable.setDbName(TO_DB_NAME);
-    when(client.getTable(FROM_DB_NAME, FROM_TABLE_NAME)).thenReturn(fromTable);
-    when(client.getTable(TO_DB_NAME, TO_TABLE_NAME)).thenReturn(toTable);
+    when(client.getTable(FROM_DB_NAME, FROM_TABLE_NAME)).thenReturn(new Table(fromTable));
```

Comment:
these test were wrong cause the "test" table object was injected into the real code and then mutated so the test weren't testing what they thought they were testing. Was never failing.